prestoadmin/standalone/config.py

```diff
@@ -140,6 +143,24 @@ def validate_workers(workers):
     return workers
 
 
+def _expand_host(host):
+    m = re.match("(.*)\[(\d{1,})-(\d{1,})\](.*)", host)
+    if m is not None and len(m.groups()) == 4:
+        prefix, start, end, suffix = m.groups()
+        if len(start) == len(end) and len(start) > 1:
+            num_fmt = "{0:0" + str(len(start)) + "d}"
+            host_list = [num_fmt.format(i) for i in range(int(start), int(end) + 1)]
+            return ["{prefix}{num}{suffix}".format(
+                prefix=prefix, num=i, suffix=suffix)
+                    for i in host_list]
```

Comment:
Yeah... but in that case I would prefer to either extract method or variable, splitting line does not help IMO